ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1971,7 +1971,14 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
     private func setupBackgroundColors() {
         if backgroundStyleType != .custom {
             automaticallyUpdatesBackgroundConfiguration = false
-            var backgroundConfiguration = UIBackgroundConfiguration.clear()
+            var backgroundConfiguration: UIBackgroundConfiguration
```

Comment:
Out of curiosity, is there any impact to the selection highlight when tapping a cell on iOS? Does it still highlight as expected within the bounds of the cell?

Reply:
i don't remember what i did when i stood this up. might be worth looking back on verification notes.. but can you double check all the different variation of popupmenucell and action cells etc? i think darkelevated colors was a lot of pain.  like mischreiber mentioned, keyboard selection will be different from selecting a cell. make sure those are properly covered.

Reply:
So far, looks good to me 
| Before | After |
| - | - |
| ![plain before](https://github.com/microsoft/fluentui-apple/assets/55368679/b02ed45a-1ef4-496c-bfa6-e3bcf80e3ed7) | ![plain after](https://github.com/microsoft/fluentui-apple/assets/55368679/a514f754-fec8-482e-bf53-4047b8412d86) |
| ![grouped before](https://github.com/microsoft/fluentui-apple/assets/55368679/8df127b1-e648-4f88-85bc-ff96239df7fb) | ![grouped after](https://github.com/microsoft/fluentui-apple/assets/55368679/c9c4f249-31d9-4b39-9b93-f94c27db71a2) |
| ![popup light before](https://github.com/microsoft/fluentui-apple/assets/55368679/5a66a81f-52ff-4a67-8c0f-69f745777c66) | ![popup light after](https://github.com/microsoft/fluentui-apple/assets/55368679/9fea5d5e-856a-4ba8-9557-5404c8db20ad) |
| ![popup dark before](https://github.com/microsoft/fluentui-apple/assets/55368679/d032bfac-d6a2-4d2a-a9a9-f496f209783a) | ![popup dark after](https://github.com/microsoft/fluentui-apple/assets/55368679/364fe31d-514c-4de7-8eac-a9ae9cc5fc9a) |
| ![other light before](https://github.com/microsoft/fluentui-apple/assets/55368679/7b1e9bea-6dee-4bb0-9800-e72b82d88185) | ![other light after](https://github.com/microsoft/fluentui-apple/assets/55368679/a7be7792-4a22-4db7-93b7-3fa19194babe) |
| ![other dark before](https://github.com/microsoft/fluentui-apple/assets/55368679/5cf4c74d-8098-4228-bbad-82c8ae07aaac) | ![other dark after](https://github.com/microsoft/fluentui-apple/assets/55368679/7c6191e5-7a45-4b2d-9e6f-0cb88d61d3d7) |
| https://github.com/microsoft/fluentui-apple/assets/55368679/0c1267c6-22a5-4e91-93a3-f7f0ce3e5f1e | https://github.com/microsoft/fluentui-apple/assets/55368679/c1ad0994-d414-4bdf-8a38-42d6ace1e999 |

Reply:
thanks for the image. i guess the only difference in my pr notes 2 years ago is that back in the day the action cells were not group style it was plain .https://github.com/microsoft/fluentui-apple/pull/1110